tools/build_defs/pkg/make_deb.py

```diff
@@ -93,16 +93,20 @@ def AddArFileEntry(fileobj, filename,
                    content='', timestamp=0,
                    owner_id=0, group_id=0, mode=0o644):
   """Add a AR file entry to fileobj."""
-  fileobj.write((filename + '/').ljust(16))    # filename (SysV)
-  fileobj.write(str(timestamp).ljust(12))      # timestamp
-  fileobj.write(str(owner_id).ljust(6))        # owner id
-  fileobj.write(str(group_id).ljust(6))        # group id
-  fileobj.write(oct(mode).ljust(8))            # mode
-  fileobj.write(str(len(content)).ljust(10))   # size
-  fileobj.write('\x60\x0a')                    # end of file entry
+  inputs = [
+    (filename + '/').ljust(16),   # filename (SysV)
+    str(timestamp).ljust(12),      # timestamp
+    str(owner_id).ljust(6),        # owner id
+    str(group_id).ljust(6),        # group id
+    oct(mode).ljust(8),            # mode
+    str(len(content)).ljust(10),   # size
+    '\x60\x0a',                    # end of file entry
+  ]
+  for i in inputs:
+    fileobj.write(i.encode())
```

Comment:
I think we should explicitly specify the encoding as ascii or utf-8, instead of leaving it ambiguous.  If we choose utf-8, then we have to encode first, then justify to 8 or 16 bytes.  It looks like this is only called with ascii filenames, though.